### PR TITLE
Fix/usdt transaction

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: src/components/SendTokens/SendTokens.js [hunk 1]
```diff
@@ -162,6 +166,9 @@ export function SendTokens(props: SendTokensProps) {
 
   const selectedTransactionFee = transactionFeeOptions.find(item => item.id === data.transactionFee);
 
+  
+  console.log('#mzm data', data);
```
Comment (andregoncalves) [inline src/components/SendTokens/SendTokens.js:170]: Stray log?

**Review by andregoncalves — APPROVED** [review]
LGTM

**Comment by andregoncalves** [comment]
@maycon-mello this is fixed right? Can be closed ?

**Comment by maycon-mello** [comment]
Yeah, this should be closed

**Pull request merged**